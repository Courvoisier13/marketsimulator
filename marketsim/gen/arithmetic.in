1 + (-6*z)
-t/p -a*l