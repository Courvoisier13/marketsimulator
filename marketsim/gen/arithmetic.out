1 + (-6*z) -> [1.11] parsed: Add(Const(1.0),Mul(Const(-6.0),Var(z)))
-t/p -a*l -> [1.10] parsed: Sub(Div(Neg(Var(t)),Var(p)),Mul(Var(a),Var(l)))
