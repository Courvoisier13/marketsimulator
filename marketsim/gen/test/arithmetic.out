1 + (-6*z) ->
Add:
- Const:
| - 1.0
- Mul:
| - Const:
| | - -6.0
| - Var:
| | - z


-t/p -a*l ->
Sub:
- Div:
| - Neg:
| | - Var:
| | | - t
| - Var:
| | - p
- Mul:
| - Var:
| | - a
| - Var:
| | - l


